project/soulrrrrr/README.rst

```diff
@@ -0,0 +1,85 @@
+========================
+Columnar Array
+========================
+
+Basic Information
+=================
+
+https://github.com/soulrrrrr/ColumnarArray
+
+Problem to Solve
+================
+
+This problem is from https://yyc.solvcon.net/en/latest/nsd/schedule/22au_nycu/schedule.html#columnar-array.
+
+Prospective Users
+=================
+
+People originally uses Array of Struct in Python and wants to speed up.
+
+System Architecture
+===================
+
+Analyze how your system takes input, produces results, provide interface, and
+performs any other operations.  Describe the system's work flow.  You may
+consider to use a flow chart but it is not required.  Specify the constraints
+assumed in your system.  Describe the modularization of the system.
+
+A class template :cpp:class:`!ColumnarArray` is holding the arrays of struct
+and the row accessor of it. The arrays of struct inside :cpp:class:`!SimpleArray`
+is :cpp:class:`!Vector`.
+
+At the moment I am planning, :cpp:class:`!ColumnarArray` can only deal with 
+structs have numeric types only.
```

Comment:
Numeric types are a good constraint to start with.

Reply:
Ok.